File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
   
public int[] twoSum(int[] nums, int target){

    int[] result = new int[2];

HashMap<Integer,Integer> map = new HashMap<>();

    for(int i = 0; i < nums.length; i++ ){
        for(int j = i+1; j < nums.length; j++ ){
            if(nums[i]+ nums[j] == target){
                result[1] = j;
                result[0] = i;

                return result;
            }
      }
    }

    return null;
    
}
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
#
# 
#
#For ever numb in
#

#return arry
    import heapq

    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freqmap = dict()
        array = []
        for i in range(len(nums)+1):
            array.append([])

        result = []
  

        for num in nums:
            if num in freqmap:
                freqmap[num] += 1
            else:
                freqmap[num] = 1
        
        for key, val in freqmap.items():
            array[val].append(key)
         
        
        i = len(array)-1
        while(k >0 or i < 0):
            for n in array[i]:
                if(array[i]):
                    result.append(n)
                    k -= 1
            i -= 1
            
        
        
           

        return result
            

        